### PR TITLE
Update warpcast.json

### PR DESCRIPTION
**Pull request by vp1337**
Update neynar JSON file with latest metadata for Bounties channel.

### CODE REVIEW

File: warpcast.json [hunk 1]
```diff
@@ -886,5 +886,11 @@
         "parent_url": "https://rarible.com",
         "image": "https://warpcast.com/~/channel-images/rarible.jpg",
         "channel_id": "rarible"
+     },
+    {
```
Comment (PirosB3) [inline warpcast.json:890]: Hey @vp1337 I'll let @manan19 take this from here, but the content looks indeed correct.

Would you mind tidying up the JSON, it looks unformatted. Thanks!

Comment (PirosB3) [reply warpcast.json:890]: image

The brackets that you put in are not aligned with the rest of the brackets

Comment (vp1337) [reply warpcast.json:890]: I edited the misalignment in Codespaces. Does it automatically edit the PR or should make additional changes?

Comment (PirosB3) [reply warpcast.json:890]: Unfortunately the brackets do not look aligned in this code :(

Comment (vp1337) [reply warpcast.json:890]: Should be good now, excuse me for the spam.

Comment (PirosB3) [reply warpcast.json:890]: It's perfect

**Pull request merged**